### PR TITLE
117 volunteer status

### LINKED ISSUE #1117: Add a new column which specifies the volunteer status for event(s) on EventList page
On eventList Page when you click on  the Volunteer Details tab, the volunteer groups(attended, wailisted, or RSVPed) can be checked and a big list of people will be shown from either of the three groups which is hard to tell who RSVPed, attended, or wailisted. 

The task here is to add a new column on a display table which specifies what specific volunteer group a volunteer belongs to, whether attended, wailisted, or rsvped. 

image

### PR DESCRIPTION
**Pull request by azabeli**
Task: On the eventList Page when you click on the Volunteer Details tab, the volunteer groups(attended, waitlisted, or RSVPed) can be checked and a big list of people will be shown from either of the three groups which is hard to tell who RSVPed, attended, or waitlisted. The task here is to add a new column on a display table that specifies what specific volunteer group a volunteer belongs to, whether attended, waitlisted, or RSVPED.

test: 

1. Go to the Event List page 
2. select the event you want to view 
3. Then click on the Volunteer Details Tab
4.  Check the Volunteer Status checkbox
5. Make sure the Event status column/Line is displayed on the Table View or the volunteer status is displayed on the card View

resolves #1117

### CODE REVIEW

File: app/static/js/slcNewProposal.js [hunk 1]
```diff
@@ -121,7 +121,6 @@ $(document).ready(function(e) {
   // -----------------------------------------
       $("#instructorTable").on("click", "#remove", function() {
         let closestRow =  $(this).closest("tr")
-        $("#instructorTableNames input[value="+closestRow.data('username')+"]").remove()
```
Comment (andersoncedu) [inline app/static/js/slcNewProposal.js]: Is this change relevant to the issue?

**Comment by andersoncedu** [comment]
A person could belong to multiple groups such as waitlist and attended or rsvp and attended, but only attended will show up.

File: app/templates/events/volunteerDetails.html [hunk 2]
```diff
@@ -39,6 +40,7 @@ <h4 class="nameSelect" nowrap><b>{{ participant.user.firstName }} {{ participant
       <td class="nameSelect" nowrap><b>{{ participant.user.firstName }} {{ participant.user.lastName }}</b></td>
       <td class="phoneSelect" nowrap>{{ participant.user.phoneNumber if participant.user.phoneNumber else "" }}</td>
       <td class="emailSelect" nowrap>{{ participant.user.email }}</td>
+      <td class="statusSelect" nowrap>{{ volunteerType|title }}</div>
```
Comment (andersoncedu) [inline app/templates/events/volunteerDetails.html]: You closed this with a /div tag.

File: app/templates/events/volunteerDetails.html [hunk 3]
```diff
@@ -133,8 +137,10 @@ <h3 style="text-align: center; width:180%;">
           <th class="text-center">Name</th>
           <th class="phoneSelect">Phone Number</th>
           <th class="emailSelect">Email</th>
+          <th class="statusSelect">Volunteer Status</th>
           <th class="dietRestrictionSelect">Dietary Restriction</th>
           <th class="emergencyContactSelect">Emergency Contact</th>
+
```
Comment (stevensonmichel) [inline app/templates/events/volunteerDetails.html]: Unnecessary extra line.

File: app/static/js/volunteerDetails.js [hunk 4]
```diff
@@ -52,6 +52,7 @@ $(document).ready(function () {
 	function getCheckBoxes() {
 		$(".displayCheckbox").each(function () {
 			let checkboxId = this.id;
+
```
Comment (stevensonmichel) [inline app/static/js/volunteerDetails.js]: Unnecessary extra line

**Comment by bledsoef** [comment]
@AndersonStettner could you take a look at Chris's comment and let me know what you think?

**Comment by AndersonStettner** [comment]
> @AndersonStettner could you take a look at Chris's comment and let me know what you think?

I think I am fine with that behavior. I imagine they would only care about the most recent attendance status and if they want to know if a person RSVPd or was on the waitlist they can look at the RSVP log.

**Review by andersoncedu — DISMISSED** [review]
Looks good to me!

**Review by Karina-Agliullova — APPROVED** [review]
Everything looks good and works as expected! We tested all the edge cases we could have thought of. All the comments were addressed. Test cases pass. Good job!

**Review by gahimbaref — APPROVED** [review]
